test/e2e/testnet/testnet.go

```diff
@@ -317,23 +331,32 @@ func (t *Testnet) Start() error {
 			genesisNodes = append(genesisNodes, node)
 		}
 	}
+	// start genesis nodes asynchronously
 	for _, node := range genesisNodes {
-		err := node.Start()
+		err := node.Instance.StartWithoutWait()
 		if err != nil {
 			return fmt.Errorf("node %s failed to start: %w", node.Name, err)
 		}
 	}
+	// wait for instances to be running
 	for _, node := range genesisNodes {
-		client, err := node.Client()
+		err := node.Instance.WaitInstanceIsRunning()
 		if err != nil {
-			return fmt.Errorf("failed to initialize node %s: %w", node.Name, err)
+			return fmt.Errorf("node %s failed to start: %w", node.Name, err)
 		}
+	}
+	// wait for nodes to sync
+	for _, node := range genesisNodes {
 		for i := 0; i < 10; i++ {
-			resp, err := client.Status(context.Background())
+			executor, err := t.GetExecutor()
 			if err != nil {
-				return fmt.Errorf("node %s status response: %w", node.Name, err)
+				return fmt.Errorf("failed to get executor: %w", err)
 			}
-			if resp.SyncInfo.LatestBlockHeight > 0 {
+			currentHeight, err := node.GetHeight(executor)
```

Comment:
Using a workaround to avoid port-forwarding which is very flaky.
This uses an instance running inside the cluster (executor) to fetch the height from the RPC.

This should be temporary and the DevOps team is working on a permanent solution for that.